packages/frontend/editor-ui/src/features/ai/chatHub/components/ChatSidebar.vue

```diff
@@ -14,11 +14,19 @@ const isMobileDevice = useMediaQuery(MOBILE_MEDIA_QUERY);
 const route = useRoute();
 const sidebar = useChatHubSidebarState();
 
+const EDGE_TRIGGER_DISTANCE = 10; // pixels from left edge to trigger sidebar
+
 watch(
 	() => route.fullPath,
 	() => uiStore.closeModal(CHAT_HUB_SIDE_MENU_DRAWER_MODAL_KEY),
 );
 
+useEventListener(window, 'mousemove', (event: MouseEvent) => {
```

Comment:
If user ends up on the `Custom Agents` page and collapses the sidebar this is the only way to get it back, and if they are on touch device like ipad there's no way to make this appear 😬 

We don't really support touch users but given the new chat users group there could be more "casual" users coming in..

Could we add the side menu button to Custom Agent's page header like we have on chat pages?

https://github.com/user-attachments/assets/c279d29e-0763-4300-ab9a-14a2d3a739f5